packages/odyssey-storybook/src/components/Banner/Banner.mdx

```diff
@@ -54,27 +50,31 @@ In addition to the required content and placement, Banners have several optional
 
 Banners can utilize an optional Title to give an at-a-glance lede.
 
-**STORY MISSING**
+<Canvas>
+  <Story id="components-banner--with-title" />
+</Canvas>
 
 ### Actions
 
-It's often useful to direct users toward an appropriate action, especially for fixing errors.
-
-**STORY MISSING**
-
-The actions section is limited to links. If it's necessary to provide the user with an action, please direct them to the appropriate flow instead of beginning a new process inline.
+It's often useful to direct users toward an appropriate action, especially for fixing errors. Please utilize <a href="?path=/docs/components-link--monochrome">Link</a> for these actions. If it's necessary to provide the user with an action, please direct them to the appropriate flow instead of beginning a new process inline.
```

Comment:
we can enforce only a specific monochrome link variant in code too BTW it will just require a tiny bit more scripting